docs/pages/versions/unversioned/sdk/storereview.md

```diff
@@ -79,3 +92,14 @@ It is important that you follow the [Human Interface Guidelines](https://develop
 - Don't call `StoreReview.requestReview()` from a button - instead try calling it after the user has finished some signature interaction in the app.
 - Don't spam the user
 - Don't request a review when the user is doing something time sensitive like navigating.
+
+### Write iOS Reviews
+
+You can redirect someone to the "Write a Review" screen for an app in the iOS App Store by using the query parameter `action=write-review`. For example:
+
+```ts
+const itunesItemId = 982107779;
+Linking.openURL(`https://apps.apple.com/app/apple-store/id${itunesItemId}?action=write-review`);
+```
+
+This can occur when the options were invalid, when the network connection is weak, or when the preview is dismissed before the content could finish loading.
```

Comment:
IMO this paragraph is out of context and does not fit the section - is it supposed to be below the example? 

Also, it may be worth mentioning, that on Android [you can redirect to Play store](https://developer.android.com/distribute/marketing-tools/linking-to-google-play) this way:
```ts
const applicationId = "com.myapp"; //in some places also called packageName, but it's not exactly the same thing
Linking.openURL(`https://play.google.com/store/apps/details?id=${applicationId}`
```